File: app/src/main/java/fm/dex/view/activity/BaseActivity.java
package fm.dex.view.activity;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;

import fm.dex.R;

abstract class BaseActivity extends AppCompatActivity {

    protected void addFragment(Fragment fragment) {
        final FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
        ft.add(R.id.content_view, fragment, fragment.getClass().getSimpleName());
        ft.commit();
    }
}

File: app/src/main/kotlin/fm/dex/view/activity/BaseActivity.kt
package fm.dex.view.activity

import android.support.v4.app.Fragment
import android.support.v7.app.AppCompatActivity

import fm.dex.R

abstract class BaseActivity : AppCompatActivity() {

    protected fun addFragment(fragment: Fragment) {
        val ft = supportFragmentManager.beginTransaction()
        ft.add(R.id.content_view, fragment, fragment.javaClass.simpleName)
        ft.commit()
    }
}
